packages/lab/src/slider/internal/SliderThumb.css

```diff
@@ -0,0 +1,56 @@
+.saltSliderThumb {
+  position: absolute;
+  width: var(--salt-size-border-strong);
+  height: var(--salt-size-selectable);
+  background: var(--salt-accent-borderColor);
+  transform: translateX(-50%);
+  cursor: pointer;
+  outline: none;
+  top: -5px;
+}
+
+.saltSliderThumb-disabled {
+  background: var(--salt-accent-borderColor-disabled);
+  pointer-events: none;
+}
+
+.saltSliderThumb-focused {
+  outline-style: var(--salt-focused-outlineStyle);
+  outline-width: var(--salt-focused-outlineWidth);
+  outline-offset: var(--salt-focused-outlineOffset);
+  outline-color: var(--salt-focused-outlineColor);
+}
+
+.saltSliderThumb:active,
+.saltSliderThumb-dragging {
+  cursor: grabbing;
+}
+
+.saltSliderThumb-input {
+  opacity: 0.0001;
+  -webkit-appearance: none;
+  appearance: none;
+  pointer-events: none;
+  left: var(--slider-progressPercentage);
+  width: var(--salt-size-border-strong);
+  height: var(--salt-size-selectable);
+  border: 0;
+  margin: 0;
+  padding: 0;
+  position: absolute;
+  cursor: pointer;
+  overflow: hidden;
+  background: 0 0;
+}
+
+.saltSliderThumb-secondThumb {
+  z-index: 10;
+}
+
+/* Visually hide accessible text. Can not
+  use visibility: hidden as we need the screen
+  reader to announce it */
+.saltSliderThumb-accessibleText {
+  position: absolute;
```

Comment:
I think we should use the standard approach here? There should be an example elsewhere in the code